doc/syntax.org

```diff
@@ -0,0 +1,43 @@
+#+TITLE:   Markup Syntax Guide for Pancacke(d)
+#+AUTHOR:  Scott Wang
+#+STARTUP: align indent
+#+OPTIONS: H:5
+
+
+* Overview
```

Comment:
This is just a plain document for the language, so not this one. Is that work for you?